pkg/generators/openapi.go

```diff
@@ -328,6 +328,7 @@ func (g openAPITypeWriter) generateMembers(t *types.Type, required []string) ([]
 			required = append(required, name)
 		}
 		if err = g.generateProperty(&m, t); err != nil {
+			glog.Errorf("Error when generating: %v, %v\n", name, m)
```

Comment:
isn't the error printed by the caller of `generateMembers`? Or could it? Printing and returning the same error always looks odd.